dcrutil/util.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2013, 2014 The btcsuite developers
+// Copyright (c) 2015-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package dcrutil
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+	"github.com/decred/dcrd/dcrec/secp256k1/v3/ecdsa"
+	"github.com/decred/dcrd/wire"
+)
+
+// VerifyMessage verifies that signature is a valid signature of message and was created
+// using the secp256k1 private key for address.
+func VerifyMessage(address string, signature string, message string, params AddressParams) (bool, error) {
```

Comment:
Why return a `bool` and an `error`.  If it's invalid, just return an error.  No error means is verified.  The caller can easily convert it to a bool if it doesn't care about the error with `valid := err == nil`.